### PR TITLE
Update service description doc

### PR DESCRIPTION
**Pull request by szwedm**
**Description**

Changes proposed in this pull request:

- fix mistakes in the doc,
- update provisioning parameters.

**Related issue(s)**

### CODE REVIEW

File: docs/03-01-service-description.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Service description
 
-Kyma Environment Broker (KEB) is compatible with the [Open Service Broker API](https://www.openservicebrokerapi.org/) (OSBA) specification. It provides a ServiceClass that provisions Kyma Runtime on a cluster.
+Kyma Environment Broker (KEB) is compatible with the [Open Service Broker API](https://www.openservicebrokerapi.org/) (OSBAPI) specification. It provides a ServiceClass that provisions Kyma Runtime on a cluster.
```
Comment (IwonaLanger) [inline docs/03-01-service-description.md:3]: It's better to use "SAP BTP, Kyma runtime", or the abbreviated name "SKR" for the managed offering and just "Kyma" for the open-source project. I suggest replacing all "Kyma Runtime"  occurrences with "SKR" after introducing the full name at the beginning. The abbreviation is already used in some sections of this doc.

Comment (IwonaLanger) [inline docs/03-01-service-description.md:3]:
```suggestion
Kyma Environment Broker (KEB) is compatible with the [Open Service Broker API](https://www.openservicebrokerapi.org/) (OSBAPI) specification. It provides a ServiceClass that provisions SAP BTP, Kyma runtime (SKR) on a cluster.
```

File: docs/03-01-service-description.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ The supported plans are as follows:
 | `gcp` | `ca6e5357-707f-4565-bbbd-b3ab732597c6` | Installs Kyma Runtime on the GCP cluster. |
 | `trial` | `7d55d31d-35ae-4438-bf13-6ffdfa107d9f` | Installs Kyma trial plan on Azure, AWS or GCP. |
 | `free` | `b1a5764e-2ea1-4f95-94c0-2b4538b37b55` | Installs Kyma free plan on Azure or AWS. |
-| `own_cluster` | `b1a5764e-2ea1-4f95-94c0-2b4538b37b55` | Installs Kyma on custom K8S cluster. |
+| `own_cluster` | `03e3cb66-a4c6-4c6a-b4b0-5d42224debea` | Installs Kyma on custom K8S cluster. |
```
Comment (IwonaLanger) [inline docs/03-01-service-description.md:18]:
```suggestion
| `own_cluster` | `03e3cb66-a4c6-4c6a-b4b0-5d42224debea` | Installs Kyma on a custom Kubernetes cluster. |
```

File: docs/03-01-service-description.md [hunk 3]
```diff
@@ -39,16 +39,19 @@ These are the provisioning parameters that you can configure:
...
 | **context.globalaccount_id** | string | Provides a global account ID for an SKR. | No | None |
-| **context.sm_platform_credentials.credentials.basic.username** | string | Provides the Service Manager username for an SKR. | No | None |
-| **context.sm_platform_credentials.credentials.basic.password** | string | Provides the Service Manager password for an SKR. | No | None |
-| **context.sm_platform_credentials.url** | string | Provides the Service Manager URL for an SKR. | No | None |
+| **context.sm_operator_credentials.clientid** | string | Provides a client ID for SAP BTP Service Operator. | No | None |
+| **context.sm_operator_credentials.clientsecret** | string | Provides a client secret for SAP BTP Service Operator. | No | None |
+| **context.sm_operator_credentials.sm_url** | string | Provides a SAP Service Manager URL for SAP BTP Service Operator. | No | None |
+| **context.sm_operator_credentials.url** | string | Provides an authentication URL for SAP BTP Service Operator. | No | None |
+| **context.sm_operator_credentials.xsappname** | string | Provides a XSApp name for SAP BTP Service Operator. | No | None |
```
Comment (IwonaLanger) [inline docs/03-01-service-description.md:46]:
```suggestion
| **context.sm_operator_credentials.xsappname** | string | Provides an XSApp name for SAP BTP Service Operator. | No | None |
```

File: docs/03-01-service-description.md (hunk 1 above)
Comment (szwedm) [reply docs/03-01-service-description.md:3]: suggestion applied.

Comment (szwedm) [reply docs/03-01-service-description.md:3]: suggestion applied.

File: docs/03-01-service-description.md (hunk 2 above)
Comment (szwedm) [reply docs/03-01-service-description.md:18]: suggestion applied.

File: docs/03-01-service-description.md (hunk 3 above)
Comment (szwedm) [reply docs/03-01-service-description.md:46]: suggestion applied.

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**